Assets/Scripts/SelectionManager.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SelectionManager : MonoBehaviour
+{
+
+    [SerializeField] private string selectableTag = "Selectable";
```

Comment:
Again, try to do use one naming convention, again I suggest startLowerCaseThenUpperForEveryWord or same with underscore at the start